nifi-nar-bundles/nifi-standard-services/nifi-hbase_1_1_2-client-service-bundle/nifi-hbase_1_1_2-client-service/src/main/java/org/apache/nifi/hbase/HBase_1_1_2_ClientService.java

```diff
@@ -171,6 +188,15 @@ protected Connection createConnection(final ConfigurationContext context) throws
             }
         }
 
+        UserGroupInformation.setConfiguration(hbaseConfig);
+
+        if (UserGroupInformation.isSecurityEnabled()) {
+            LOG.info("SECURITY IS ENABLED");
+            UserGroupInformation.loginUserFromKeytab(context.getProperty(KERBEROS_PRINCIPAL).getValue(),
```

Comment:
Ok so looking at UGI it seems to spawn a kerberos credential renewal thread. I think that maybe will not work for processors because of thread suspension. However this is not a processor it is a service. Does anyone know if "services" once enabled ever suspend threads? If not than we may be safe letting UGI handle the renewal of kerberos credentials.